parser/parser.go

```diff
@@ -5,13 +5,16 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
+	"path/filepath"
 	"regexp"
 
 	"github.com/dogtools/dog/types"
 	"github.com/ghodss/yaml"
 )
 
 var ErrMalformedStringArray = errors.New("Malformed strings array")
+var ErrNoDogfiles = errors.New("No dogfiles found")
```

Comment:
I get the feeling that this idea of the different Dogfiles being treated as one comes from when they were literally concatenated together before parsing; I'm not sure how useful it is now.